assets/src/hooks/DagreHook.ts

```diff
@@ -21,14 +21,17 @@ const DagreHook = {
     const graph = createDagre(this.el, width, height);
     this.graph = graph;
 
+    let hasInteracted = false;
```

Comment:
Why don't you put that in inside the mounted hook instance? `this.hasInteracted = false`

Reply:
Sure, done in 3aa9e1c